compiler/noirc_driver/src/lib.rs

```diff
@@ -66,10 +66,16 @@ pub struct CompileOptions {
     #[arg(long = "force")]
     pub force_compile: bool,
 
-    /// Emit debug information for the intermediate SSA IR
+    /// Emit debug information for the intermediate SSA IR to stdout
     #[arg(long, hide = true)]
     pub show_ssa: bool,
 
+    /// Emit the unoptimized SSA IR to file.
+    /// The IR will be dumped into the workspace target directory,
+    /// under the name of the package that was compiled.
```

Comment:
With `ssa-ir` appended to the name of the package:
```suggestion
    /// under [package-compiled]ssa-ir.ssa.json
```
(if I got that `join`/`with_extension` chain right)

Reply:
Hmm, do we really need both a "ssa-ir" suffix and a `.ssa` then? I vote we have only the later

Reply:
Yeah I think I added the `.ssa` without removing the `ssa-ir` suffix. Going to just keep `.ssa`

Reply:
As I changed the `join` chain here I manually changed this comment to the following:
```
    /// Emit the unoptimized SSA IR to file.
    /// The IR will be dumped into the workspace target directory,
    /// under `[compiled-package].ssa.json`.
```